Automattic/jetpack
# Map block: Fix map zoom being reset after marker color change

Fixes https://github.com/Automattic/wp-calypso/issues/76290

## Proposed changes:
This PR fixes the map zoom being reset after marker color change


| Before | After |
|-|-|
| ![CleanShot 2023-05-05 at 13 10 13](https://user-images.githubusercontent.com/528287/236444097-b50aa0f9-50a4-4a44-aac5-063423faf5c7.gif) |  ![CleanShot 2023-05-05 at 13 13 38](https://user-images.githubusercontent.com/528287/236444130-b1ee0a08-fd25-4eda-9d3a-8992df63d527.gif) |

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






1. Apply this PR (either by running Jetpack locally or using the JP downloader to your sandbox)
2. If running locally enable Mapkit by setting the cookie (`document.cookie = 'map_provider=mapkit';`)
3. Add a new post
4. Add a map with one marker
5. Zoom out a bit
6. Change the marker color (in the block settings in the right)
7. The map zoom should stay the same
